Smarty/templates/modules/Calendar4You/CalendarView.tpl

```diff
@@ -456,6 +456,23 @@ function hideITSEventInfo(){
     jQuery('#event_info_content').html('');
 
 }
+
```

Comment:
Yes that was my first thought as well. Unfortunately the 'fullcalendar' plugin needs to refire to calculate widths in this case (widths are not relative but calculated absolutely). Another solution would maybe be to wrap the entire content, make it 120% of the screen size and shift it left and right via jQuery.